receiver/k8sclusterreceiver/internal/collection/collector.go

```diff
@@ -69,6 +69,7 @@ func (dc *DataCollector) CollectMetricData(currentTime time.Time) pmetric.Metric
 		if crm.ScopeMetrics().Len() > 0 {
 			crm.MoveTo(customRMs.AppendEmpty())
 		}
+		node.RecordMetrics(dc.metricsBuilder, o.(*corev1.Node), ts)
```

Comment:
I think this is ok since the old metric can be disabled by setting it to `[]`.  We should document somewhere that if you use the new metric you should disable the old one.